Source/ARTURLSessionServerTrust.m

```diff
@@ -20,7 +20,29 @@ @implementation ARTURLSessionServerTrust
 - (instancetype)init:(dispatch_queue_t)queue {
     if (self = [super init]) {
         _queue = queue;
-        _session = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration] delegate:self delegateQueue:nil];
+        NSURLSessionConfiguration *config = [NSURLSessionConfiguration ephemeralSessionConfiguration];
+#if TARGET_OS_IOS
```

Comment:
What do you think about this one instead of all this statements?
```suggestion 0+21
        if(@available(iOS 18.5, macOS 10.12, tvOS 13.0, *)) {
            config.TLSMinimumSupportedProtocolVersion = tls_protocol_version_TLSv12;
            } else {
            config.TLSMinimumSupportedProtocol = kTLSProtocol12;
        }
```